x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/EsqlActionBreakerIT.java

```diff
@@ -105,6 +114,7 @@ protected EsqlQueryResponse run(EsqlQueryRequest request) {
                 return resp;
             }
         }
+        return super.run(request);
```

Comment:
Not sure I understand why we're running the request twice here now. Maybe a comment would help.

Reply:
I added a comment in https://github.com/elastic/elasticsearch/pull/121942/commits/8bb4cfd4624284cd70888c465f3ec8dfe3c3ddbf